lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -122,6 +122,16 @@ class GoogleCloudOutput < BufferedOutput
     # Cloud Logging API.
     config_param :use_grpc, :bool, :default => false
 
+    # Whether to allow non-UTF-8 characters in user logs. If set to true, any
+    # non-UTF-8 character would be replaced by the string specified by
+    # 'non_utf8_replacement'. If set to false, any non-UTF-8 character would
```

Comment:
Changed for all 4 occurences. And I just built and tested the plugin on a VM again.